dags/vm_resource.py

```diff
@@ -133,3 +136,5 @@ class DockerImage(enum.Enum):
   XPK_MAXTEXT_TEST = "gcr.io/tpu-prod-env-multipod/xpk_maxtext_test:latest"
   MAXTEXT_JAX_STABLE = f"gcr.io/tpu-prod-env-multipod/maxtext_jax_stable:{datetime.datetime.today().strftime('%Y-%m-%d')}"
   MAXTEXT_JAX_NIGHTLY = f"gcr.io/tpu-prod-env-multipod/maxtext_jax_nightly:{datetime.datetime.today().strftime('%Y-%m-%d')}"
+  MAXTEXT_GPU_JAX_STABLE = f"gcr.io/tpu-prod-env-multipod/maxtext_gpu_jax_stable:{datetime.datetime.today().strftime('%Y-%m-%d')}"
```

Comment:
Hi @jonb377, GCR is going to deprecate soon, does your team plan to migrate those to AR soon?

Reply:
I saw that as well, it looks like it's not going to shut down support for projects already using GCR: https://cloud.google.com/container-registry/docs/deprecations/container-registry-deprecation